MSMetaEnhancer/libs/Annotator.py

```diff
@@ -9,10 +9,13 @@ class Annotator:
     """
     Annotator is responsible for annotation process of single spectra.
     """
-    def __init__(self, converters):
-        self.converters = converters
+    def __init__(self):
+        self.converters = None
```

Comment:
Is an annotator without converters actually functional? If not, then the parameter should I think still be in the constructor.